crates/committed/src/main.rs

```diff
@@ -232,7 +232,11 @@ fn run() -> proc_exit::ExitResult {
             std::fs::read_to_string(path).to_sysexits()?
         };
         let text = trim_commit_file(&text);
-        failed |= checks::check_message(path.as_path().into(), text, &config, report)
+        let text: String = text
+            .split('\n')
+            .filter(|line| !line.trim_start().starts_with('#'))
+            .collect();
```

Comment:
I think the logic makes sense to put in `trim_commit_file`, as it does exactly that. However, since this allocates a string this would require to change the function signature (I think?).

Should this be moved elsewhere?

Reply:
What about the verbose commit trailer? I've only glossed over it but it seems this would remove the marker?

Reply:
I'd pull it out as a constant and if you see that, stop stripping comments.